pages/common/docker-secret.md

```diff
@@ -0,0 +1,24 @@
+# docker secret
+
+> Manage Docker swarm secret.
+> More information: <https://docs.docker.com/engine/reference/commandline/secret/>.
+
+- Create a new secret from stdin:
+
+`printf {{secret_content}} | docker secret create {{secret_name}} -`
```

Comment:
This could potentially be the output of any command, so it might be better to use a more generic example here.
```suggestion
`{{command}} | docker secret create {{secret_name}} -`
```